_posts/2020-04-07-trying-out-julia.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: Trying out Julia programming language
+author: hnybom
+excerpt: >
+  This post is about the Julia programming language and my journey starting from zero knowledge to, well, some knowledge of the language. So if you, like me, are interested trying out Julia language keep on reading. If you are already familiar with the subject you can safely skip this one.
+tags:
+ - Julia
+---
+
+## Let's try out Julia
+
+Few forewords. I have never tried Julia programming language before but I've read couple of headlines about it. The main motivations to me are
```

Comment:
Few forewords. (Is not a sentance). As forewords, I have never.. Voi myös jättää oikein luontevasti poiskin.

Olet lukenut muutaman otsikon juliasta? Ymmärrän, mitä haet takaa, mutta tämä ei ehkä ihan kommunikoi enkuksi.